frontend/src/metabase-types/store/dashboard.ts

```diff
@@ -10,16 +10,29 @@ import { ParameterValueOrArray } from "metabase-types/types/Parameter";
 
 export type DashboardSidebarName =
   | "addQuestion"
+  | "action"
```

Comment:
Not related to tabs but this was missing from the type which caused errors elsewhere.